### PR TITLE
pull ucsd majors for autocomplete

### PR DESCRIPTION
**Pull request by StoneT2000**
Screen Shot 2020-09-28 at 1 53 09 PM
major change is making major field longer for majors with longer names

custom major names are not possible anymore, instead users can put in "other"

### CODE REVIEW

**Comment by leestanley** [comment]
image
The div isn't tall enough on my screen to fit the ACM logo and has very little padding on the `Already have an account? Sign in.`

File: src/components/RegisterForm/majors.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
...dies — Philosophy","Italian Studies","Japanese Studies","Jewish Studies","Latin American Studies","Latin American Studies with a Concentration in Mexico","Latin American Studies with a Concentration in Migration and Border Studies","Language Studies","Linguistics","Literature/ Writing","Literatures in English","Spanish Literature","World Literature and Culture","Mathematics","Mathematics (Applied)","Mathematics — Applied Science","Mathematics — Computer Science","Probability & Statistics","Aerospace Engineering","Environmental Engineering","Mechanical Engineering","Interdisciplinary Computing and the Arts","Music","Music Humanities","Nanoengineering","Philosophy","General Physics","Physics","Physics-Biophysics","Political Science","Psychology","Business Psychology","Study of Religion","Russian, East European and Eurasian Studies","Earth Sciences","Marine Biology","Oceanic and Atmospheric Sciences","Sociology","Structural Engineering","Dance","Theatre","Third World Studies","Undeclared","Undeclared — Humanities/ Arts","Undeclared — Physical Sciences","Undeclared — Social Sciences","Urban Studies and Planning","Real Estate and Development","Visual Arts","Speculative Design", "Other"]
```
Comment (leestanley) [inline src/components/RegisterForm/majors.json:2]: Can we do some more condensing here by moving all the specializations of `International Studies: xyz` to just `International Studies`. Same with Undeclared/Bioengineering

**Review by leestanley — CHANGES_REQUESTED** [review]

**Comment by leestanley** [comment]
Not part of this PR but not sure why the year defaults to `0` here 👀 
image

**Comment by steets250** [comment]
It defaults to 0 cause the default value of that field isn't set. If we assign the first year in the array, then the drop down will work correctly. Idk if you want to include that in this PR, but it's like a 1-2 line fix.

**Comment by steets250** [comment]
Brushing the dust off this one... can you add a hint to the field so it says "Major" before a major is selected, and then switch the major field to being a full width field? It was fine with the old design but the redesign shrunk the width of the field and now the names get cut off:

Screen Shot 2020-12-17 at 5 11 23 PM

**Comment by seanye24** [comment]
Moved year field to new line, added major placeholder
#363

**Review by steets250 — APPROVED** [review]
@seanye24  looks good! You can go ahead and merge.

**Pull request merged**